### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Rudloff** [Limegrass/Wanicchou#12]
Hello,

I'm trying to build v0.1.0-beta for F-Droid (#10).
`gradle assemble` seems to work correctly but then the app crashes when I launch it on my phone:
```
07-05 16:31:23.317 28649 28649 D AndroidRuntime: Shutting down VM
07-05 16:31:23.318 28649 28649 E AndroidRuntime: FATAL EXCEPTION: main
07-05 16:31:23.318 28649 28649 E AndroidRuntime: Process: com.waifusims.wanicchou, PID: 28649
07-05 16:31:23.318 28649 28649 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.waifusims.wanicchou/com.waifusims.wanicchou.SearchActivity}: java.lang.RuntimeException: Cannot create an instance of class data.room.notes.NoteViewModel
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.app.ActivityThread.access$900(ActivityThread.java:150)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:148)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:5417)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: Caused by: java.lang.RuntimeException: Cannot create an instance of class data.room.notes.NoteViewModel
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.arch.lifecycle.ViewModelProvider$AndroidViewModelFactory.create(ViewModelProvider.java:201)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.arch.lifecycle.ViewModelProvider.get(ViewModelProvider.java:134)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.arch.lifecycle.ViewModelProvider.get(ViewModelProvider.java:102)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at com.waifusims.wanicchou.SearchActivity.onCreate(SearchActivity.java:88)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:6285)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	... 9 more
07-05 16:31:23.318 28649 28649 E AndroidRuntime: Caused by: java.lang.NoSuchMethodException: <init> [class android.app.Application]
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at java.lang.Class.getConstructor(Class.java:528)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at java.lang.Class.getConstructor(Class.java:492)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	at android.arch.lifecycle.ViewModelProvider$AndroidViewModelFactory.create(ViewModelProvider.java:199)
07-05 16:31:23.318 28649 28649 E AndroidRuntime: 	... 15 more
```

(I am using Android 6.0.1.)

**Comment by Limegrass** [comment]
Hello Rudloff,
Thanks for the help! I'm sorry, I did not properly test installation with apk for the release.
I've resolved the issue locally on my device/AVDs in v0.1.1-beta (https://github.com/Limegrass/Wanicchou/releases/download/v0.1.1-beta/wanicchou-v0.1.1-beta.apk), let me know if the crash persists for you.

**Comment by Rudloff** [comment]
v0.1.1-beta does not crash anymore. Thanks!
